src/NuGet.Clients/NuGet.VisualStudio.Implementation/Resources/VsResources.resx

```diff
@@ -179,4 +179,7 @@ The string representation of the user's input framework name.</comment>
   <data name="Error_ProjectNotInCache" xml:space="preserve">
     <value>Project {0} does not exist in the project system cache.</value>
   </data>
+  <data name="Error_InvalidPackageRefProjectCast" xml:space="preserve">
+    <value>The project '{0}' does not support PackageReference.</value>
```

Comment:
nit: single quotes are legal path characters. So they're not the best quotes to use for paths. Double-quotes are better.